README.md

```diff
@@ -1,109 +1,162 @@
+## At a Glance
+- Pair, [stage 2](https://github.com/Ada-Developers-Academy/pedagogy/blob/master/rule-of-three.md#stage-2) project
+- Due EOD Friday at 6 PM on **DATE HERE**
+- Submit this project with a PR
 
-This is a pair project. You and your partner should use all the techniques we've learned so far to keep yourselves organized and on track, and ensure that no requirements slip through the cracks.
+## Introduction
 
-This project is due: **Due Date goes here**
+[Video rental stores](https://en.wikipedia.org/wiki/Video_rental_shop) are businesses that need to track a lot of things. Namely, they need to keep track of:
+
+- the store's inventory of videos available for rental
+- their customers
+
+A small video rental business wants to start using software to help manage this data. They've already documented their video and customer information into files. Now, they're asking our team to build an API for them!
 
 ## Learning Goals
-Upon completing this project, students should be able to:
 
-- Build an ERD and set up ActiveRecord models for a given dataset / use-case
-- Expose database contents through a web API
-- Respond reasonably to bad user data in the context of an API
-- Verify the correctness of an API using controller tests
+- Design an API in Rails that is compatible with a given dataset, and design an ERD and ActiveRecord models
+- Build an API that exposes database contents
+- Apply best practices to handle edge-cases in the context of an API, and reasonably handle bad user data
+- Practice testing APIs through writing controller tests
+
+## Objective
+
+We will make an API in Rails for a small video rental store.
+
+This API will be able to serve information about the store's inventory of videos and customer information. The API will also be able to update the status of rental.
+
+You and your team should use all the techniques we've learned so far to keep yourselves organized and on track, and ensure that no requirements slip through the cracks.
+
+### Project Structure
+
+In the **"Testing Requirements"** section, there is a description of the smoke tests required for this project.
 
-This is a [stage 2](https://github.com/Ada-Developers-Academy/pedagogy/blob/master/classroom/rule-of-three.md#stage-2) project.
+In the **"Setup Requirements"** section, there is Wave 0, which focuses on creating the design of this project and setting up the Rails app. It also describes the database seeds script provided in this project.
 
-## Success Criteria
+In the **"Functional Requirements"** section, Waves 1 and 2 will focus on describing the required endpoints for this project. These endpoints are described more-or-less in order of complexity. It's recommend to build them in that order. TODO: Every endpoint must serve JSON data, and must use HTTP response codes to indicate the status of the request.
```

Comment:
Just not sure what the TODO is here for.

Reply:
Thanks for the catch! I'll address this